account_chatter/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
It should be LGPL. This must be fixed in all headers. I'd recommend just removing headers, as the manifest already says LGPL.